tutorials/Tutorial1_Basic_QA_Pipeline.ipynb

```diff
@@ -321,7 +321,7 @@
    "outputs": [],
    "source": [
     "# Alternative:\n",
-    "# reader = TransformersReader(model=\"distilbert-base-uncased-distilled-squad\", tokenizer=\"distilbert-base-uncased\", use_gpu=-1)"
+    "# reader = TransformersReader(model_name_or_path=\"distilbert-base-uncased-distilled-squad\", tokenizer=\"distilbert-base-uncased\", use_gpu=-1)"
```

Comment:
I think we are missing the same change as here in the plain python version:  Tutorial1_Basic_QA_Pipeline.py and Tutorial3_Basic_QA_Pipeline_without_Elasticsearch.py

Reply:
Thanks, I should have relied on grep instead of IDE search :)